docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -177,7 +168,7 @@ We created the `lastorder` Function and subscribed to the `order.received.v1` ev
 
    ```bash
    cloudevents send http://localhost:3000/publish \
-      --type sap.kyma.custom.myapp.order.received.v1 \
+      --type order.received.v1 \
```

Comment:
did you test that this command works ?

Reply:
I will do testing once EPP and Busola changes are done.